<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>mypopup</class>
 <widget class="QWidget" name="mypopup">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>142</width>
    <height>438</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="widget" native="true">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>10</y>
     <width>102</width>
     <height>404</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QLabel" name="label">
      <property name="text">
       <string>store location</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QPushButton" name="m_pushButton_storelocation">
      <property name="text">
       <string>...</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_2">
      <property name="text">
       <string>container</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_comboBox_container">
      <property name="layoutDirection">
       <enum>Qt::LeftToRight</enum>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_3">
      <property name="text">
       <string>audio byte rate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_comboBox_audiorate"/>
    </item>
    <item>
     <widget class="QLabel" name="label_4">
      <property name="text">
       <string>video frate</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_comboBox_videofrate"/>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="text">
       <string>video res</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_comboBox_videores"/>
    </item>
    <item>
     <widget class="QLabel" name="label_6">
      <property name="text">
       <string>video codec</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_comboBox_video_codec"/>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="text">
       <string>audio codec</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QComboBox" name="m_comboBox_audio_codec"/>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
